domain/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
the same here, lets have only one gitignore

Reply:
done